I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 646, in script call
    "Nous sommes en 1870, à Tokyo, capitale nouvellement renommée de l’Empire du Japon. Depuis peu, le pays s’ouvre aux techniques étrangères, et de nombreux occidentaux parcourent ses rues."
  File "renpy/common/00console.rpy", line 765, in script call
    python in _console:
ScriptError: could not find label 'ran_ch3_bon'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 646, in script call
    "Nous sommes en 1870, à Tokyo, capitale nouvellement renommée de l’Empire du Japon. Depuis peu, le pays s’ouvre aux techniques étrangères, et de nombreux occidentaux parcourent ses rues."
  File "renpy/common/00console.rpy", line 765, in script call
    python in _console:
  File "<string>", line 1, in script
  File "C:\renpy\renpy\ast.py", line 1620, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\renpy\renpy\script.py", line 873, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'ran_ch3_bon'.

Windows-8-6.2.9200
Ren'Py 7.1.0.882
Meiji 1.0
Mon Oct 29 12:02:58 2018
